package.json

```diff
@@ -24,21 +24,16 @@
   },
   "scripts": {
     "compile": "tsc -p .",
+    "typecheck": "tsc -p . --noEmit",
     "build": "yarn compile && cp ./boilerplate/.gitignore ./boilerplate/.gitignore.template",
-    "format": "yarn format:js && yarn format:json && yarn format:md && yarn format:ts;",
-    "format:js": "prettier --write '**/*.js'",
-    "format:json": "prettier --write '**/*.json'",
-    "format:md": "prettier --write '**/*.md'",
-    "format:ts": "prettier --write '**/*.ts{,x}'",
-    "lint": "eslint src test --ext .ts --fix && yarn format",
```

Comment:
I would expect this based on the name just to run eslint, not also run a side effect of formatting files with prettier